filebeat/module/system/syslog/ingest/pipeline.json

```diff
@@ -1,41 +1,48 @@
 {
-	"description": "Pipeline for parsing Syslog messages.",
-	"processors": [
-		{
-			"grok": {
-				"field": "message",
-				"patterns": [
-					"%{SYSLOGTIMESTAMP:system.syslog.timestamp} %{SYSLOGHOST:system.syslog.hostname} %{DATA:system.syslog.program}(?:\\[%{POSINT:system.syslog.pid}\\])?: %{GREEDYMULTILINE:system.syslog.message}",
-					"%{SYSLOGTIMESTAMP:system.syslog.timestamp} %{GREEDYMULTILINE:system.syslog.message}"
-				],
-        "pattern_definitions" : {
-          "GREEDYMULTILINE" : "(.|\n)*"
+    "description": "Pipeline for parsing Syslog messages.",
+    "processors": [
+        {
```

Comment:
Seems like indentation was off before: https://github.com/elastic/beats/blob/master/.editorconfig#L15 Nothing to do here, just a note. Thanks for doing the formatting.